single_tech_samples/storage/sample1_data_lifecycle/terraform/modules/services/datalifecycle/main.tf

```diff
@@ -0,0 +1,144 @@
+terraform {
```

Comment:
As mentioned in the previous comment, promote this and other files to the `terraform` folder directly and remove the (sub)directory `modules/*`.